samples_project/Assets/SampleViewer/Samples/Measure/readme.md

```diff
@@ -2,13 +2,19 @@
 
 Draw lines on the map and show distance.
 
-![Measure](Measure.PNG)
+![Measure](measure.PNG)
+
+## How to use the sample
+
+1. Set your API Key in the ArcGISMapActor using the World Outliner window (if you are using the SampleViewerLevel you can also set the key through the UI or the level blueprint).
+2. While holding shift, click on the map to begin measuring. Continue to click on the map to add additional vertices. The accumulated distance will be shown. This sample is only set up to work with a mouse and keyboard.
 
 ## How it works
 
-1. Set your API Key.
-2. While holding shift, click on the map to begin drawing. Keep clicking to add vertices. The accumulated geodetic distance will be shown. The distance calculation uses the [ArcGISGeometryEngine](https://developers.arcgis.com/unity/api-reference/gameengine/geometry/arcgisgeometryengine#distancegeodetic). This sample is only setup to work with mouse and keyboard. 
+1. This sample uses [ArcGISGeometryEngine](https://developers.arcgis.com/unity/api-reference/gameengine/geometry/arcgisgeometryengine#distancegeodetic) to calculate distance. 
+2. The measured distance is [geodetic/geodesic distance](https://pro.arcgis.com/en/pro-app/2.8/tool-reference/spatial-analyst/geodesic-versus-planar-distance.htm). Geodetic distance is calculated in a 3D spherical space as the distance across the curved surface of the world.
```

Comment:
Since the sample doesn't use Pro, I think it's better to use these 2 URLs for the links. 

https://developers.arcgis.com/documentation/glossary/geodesic/
https://developers.arcgis.com/documentation/glossary/geodetic-measurement/

The URL you added could be in also but add it as `you can find more information about the differences between geodesic and planar distance in ArcGIS Pro documentation` and use this link to land on the specific section. (The page is about the tool on ArcGIS Pro.)

https://pro.arcgis.com/en/pro-app/2.8/tool-reference/spatial-analyst/geodesic-versus-planar-distance.htm#ESRI_SECTION1_A0F81D4CC1C14618B64BEF3469196C69